desktop/core/src/desktop/hue_curl.py

```diff
@@ -0,0 +1,75 @@
+# Licensed to Cloudera, Inc. under one
```

Comment:
This is duplicated with https://github.com/cloudera/hue/blob/master/tools/ops/script_runner/lib/hue_curl.py? Maybe update the import statement?

Reply:
Wanted this to be in the Hue's default location.  Will be removing these unneeded files, once all of the informations are moved which are used.

There are still some more scripts which we use, going over them and then will sort these out.